doc/conf.py

```diff
@@ -59,15 +59,15 @@
 
 # General information about the project.
 project = 'garnett'
-copyright = '2015-2019, The Regents of the University of Michigan'
+copyright = '2015-2020, The Regents of the University of Michigan'
```

Comment:
Actually, all the files with the license agreement still have 2019. Can you fix that as part of this PR?